pkg/apis/v1alpha1/awsnodetemplate.go

```diff
@@ -104,3 +111,12 @@ type AWSNodeTemplateList struct {
 	metav1.ListMeta `json:"metadata,omitempty"`
 	Items           []AWSNodeTemplate `json:"items"`
 }
+
+// Placement contains the configuration for LaunchTemplates to run instances on dedicated hosts
+// https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-launchtemplate-launchtemplatedata.html#cfn-ec2-launchtemplate-launchtemplatedata-placement
+type Placement struct {
+	// +optional
+	HostResourceGroupArn string `json:"hostResourceGroupArn,omitempty"`
```

Comment:
casing nit and I think these should be pointers

```suggestion
	HostResourceGroupARN *string `json:"hostResourceGroupARN,omitempty"`
```